test/test_transforms_v2.py

```diff
@@ -99,7 +99,7 @@ def _script(obj):
         return torch.jit.script(obj)
     except Exception as error:
         name = getattr(obj, "__name__", obj.__class__.__name__)
-        raise AssertionError(f"Trying to `torch.jit.script` '{name}' raised the error above.") from error
+        raise AssertionError(f"Trying to 'torch.jit.script' {name} raised the error above.") from error
```

Comment:
Let's revert these changes if you don't mind: they are not strictly related to the changes in this PR and thus it makes it a bit harder to do the review as they add noise.
(It's also a matter of preference whether backticks should be used, so it's best to leave that discussion for elsewhere).

Please revert other similar changes as well

Reply:
Sure I can do that! Do you have advice on how to avoid getting flake errors from formatting the commit? I added these changes originally because flake would fail due to inclusion of the backticks.

Reply:
Hm yeah, I see (a newer version of?) `flake8`  being unhappy locally. It seems like a bug/regression to me. backticks used to be a proper python construct in Python2 and they were dropped in Python 3 in favor of `repr()`, so `flake8` is telling us to use `repr()` instead. But here we're using backticks within a string - there's no reason to error on that, so let's just ignore it.

Reply:
Sounds good!